src/classes/modules/java.base/java/util/Calendar.java

```diff
@@ -0,0 +1,3279 @@
+package java.util;
```

Comment:
Add the license file header for any files adapted from OpenJDK.

Reply:
GNU GPL v2 license file has been added here, Cyrille